cmd/rook/block-list_test.go

```diff
@@ -58,8 +59,14 @@ func TestListBlockImages(t *testing.T) {
 	out, err := listBlocks(mockRBDSysBusPath, c, e)
 	assert.Nil(t, err)
 
-	expectedOut := "NAME       POOL      SIZE       DEVICE    MOUNT\n" +
-		"myimage1   mypool1   1.00 KiB   rbd5      /tmp/mymount1\n"
+	var expectedOut string
+	if runtime.GOOS == "linux" {
+		expectedOut = "NAME       POOL      SIZE       DEVICE    MOUNT\n" +
```

Comment:
to reduce duplication, you could use the same base string, then simply append the mount path to the expectedOut